sti-kbproxy/src/main/java/uk/ac/shef/dcs/kbproxy/sparql/SPARQLProxy.java

```diff
@@ -781,11 +783,43 @@ private SelectBuilder createFulltextQueryBuilder(String content, Integer limit,
     return builder;
   }
 
+  private String createFulltextExpression(String content) {
```

Comment:
It strips the searched value of all special characters and creates a proper search expression from the individual words. I have found little information about the fulltext catalog used in Virtuoso, but it seems to accept the standard notation from SQL. Example:
```
Girls-Treff "Insel"
```
is changed into
```
'"Girls" AND "Treff" AND "Insel"'
```